apps/api/src/domain/models/audio-item/entities/audio-item.entity.ts

```diff
@@ -105,6 +107,15 @@ class AudioItemBase extends Resource implements IRadioPublishableResource {
         return this.name;
     }
 
+    translateName(textItem: MultilingualTextItem): ResultOrError<AudioItem> {
```

Comment:
Before you merge down, I'll help you find a way to reuse this logic between other aggregate root classes.